src/Expr.cc

```diff
@@ -4294,13 +4294,15 @@ InExpr::InExpr(ExprPtr arg_op1, ExprPtr arg_op2)
 
 	if ( op1->GetType()->Tag() == TYPE_PATTERN )
 		{
-		if ( op2->GetType()->Tag() != TYPE_STRING )
+		if ( op2->GetType()->Tag() == TYPE_STRING || op2->GetType()->Tag() == TYPE_TABLE )
 			{
```

Comment:
I might be missing something here - could you perhaps explain which case this covers?

Reply:
ah, ok, I think I got it. Let me just try one thing...